README.md

```diff
@@ -1 +1,294 @@
-# budget_app
\ No newline at end of file
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+<<<<<<< HEAD
+# 📖 [Piercel_payments] <a name="about-project"></a>
+
+
+
+**[Piercel_payments]** is a mobile web application where you can manage your budget and you can easily create an account and login to start managing your budget. A user can create groups to keep track of specific expenses and add transactions to those groups to stay on top of your spending. With this app, you can stay in top of list when coming up with financial plan.
+=======
+
+**[Piercel Budget]** is a mobile web application where you can manage your budget and you can easily create an account and login to start managing your budget. A user can create categories to keep track of specific expenses and add transactions to those groups to stay on top of the spending. With this app, you can stay in top of list when coming up with financial plan. This app is deployed online and you can check it.
+>>>>>>> 1036eb2 (Add the required changes)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<<<<<<< HEAD
+<details>
+  <summary>Rails is a full-stack framework. It ships with all the tools needed to build amazing web apps on both the front and back end.</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+=======
+
+<details>
+  <summary>Rails is a full-stack framework. It ships with all the tools needed to build amazing web apps on both the front and back end.</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">RubyOnRails</a></li>
+>>>>>>> 1036eb2 (Add the required changes)
+  </ul>
+</details>
+
+<details>
```

Comment:
- [ ] [OPTIONAL] You may accidentally pushed unresolved conflict to your repo, Kindly make sure to remove the conflict messages from your README